libs/ui/src/printed_ballot.tsx

```diff
@@ -160,6 +160,7 @@ function CandidateContestResult({
           </Text>{' '}
           {candidate.partyId &&
             `/ ${
+              /* istanbul ignore next: This .find should always return something */
               election.parties.find((p) => p.id === candidate.partyId)?.name
```

Comment:
I think libs/utils/find provides a version of `find` that always returns something or throws an error